entc/integration/ent/schema/student.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019-present Facebook Inc. All rights reserved.
+// This source code is licensed under the Apache 2.0 license found
+// in the LICENSE file in the root directory of this source tree.
+
+package schema
```

Comment:
Edge-schemas are tested in `integration/edgeschema`. See this example test: https://github.com/ent/ent/blob/master/entc/integration/edgeschema/edgeschema_test.go#L184

I prefer to keep edge-schema tests on the same directory, and reuse existing schemas if possible.